gui/src/app/tinystan/index.ts

```diff
@@ -39,6 +39,17 @@ interface WasmModule {
 const NULL = 0 as ptr;
 const NULLSTR = 0 as cstr;
 
+
+const HMC_SAMPLER_VARIABLES = [
+  "lp__",
+  "accept_stat__",
+  "stepsize__",
+  "treedepth__",
+  "n_leapfrog__",
+  "divergent__",
+  "energy__",
+];
```

Comment:
We don't expose anything through this interface that would cause it to use a different sampler with different default variables, right?

Reply:
Nope. If we want to do e.g. Pathfinder later we would need a different list of these algorithmic parameters, but we'd also have that behind a different function than `.sample()`